src/main/java/org/veupathdb/lib/gradle/container/tasks/jaxrs/JaxRSPatchEnumValue.kt

```diff
@@ -4,26 +4,17 @@ import org.veupathdb.lib.gradle.container.tasks.base.JaxRSSourceAction
 
 import java.io.*
 import java.util.Arrays
-import java.util.Locale
 
 open class JaxRSPatchEnumValue : JaxRSSourceAction() {
 
   companion object {
     const val TaskName = "jaxrs-patch-enums"
 
-    private const val OldLeader = "  private "
-    private const val NewLeader = "  public final "
+    private const val OldNameField = "  private String name;"
+    private const val NewNameField = "  public final String value;"
```

Comment:
Why public here?  If we have the getter, it could still be private, right?  Also wonder if even this is more than we need.  You could just write:
`public String getValue(){ return name; }`
rather than change the name of the private field.

Reply:
1. I will update it so the `value` field is not made public.  I did that out of habit but it's unnecessary here.
2. I felt that `name` was misleading.  Like you said, Java already has a `name` function on enum types, and having the property also named `name` just seems misleading.